internal/models/user.go

```diff
@@ -698,6 +698,17 @@ func (u *User) SoftDeleteUser(tx *storage.Connection) error {
 	return nil
 }
 
+func (u *User) LogoutAllSessionsExceptMine(tx *storage.Connection) error {
+	session, err := FindSessionByUserID(tx, u.ID)
```

Comment:
How do we know if the session returned here is the same one that the user is logged in to? seems like `FindSessionByUserID` will return the very first session created by the user. Also, it doesn't seemed to be used anywhere previously except in the tests

Reply:
You're right, it does return the first session created by user. It is also used only in the tests - we can move FindSessionByUserID to be within the tests as a helper if that'd be better.